src/guide/typescript/composition-api.md

```diff
@@ -50,6 +50,20 @@ const props = defineProps<Props>()
 </script>
 ```
 
+This also works if `Props` is imported from an external source. 
+
+```vue
+<script setup lang="ts">
+import type { Props } from './foo'
+
+const props = defineProps<Props>()
+</script>
+```
+
+:::info
```

Comment:
**suggestion**: we're trying to avoid the excessive usage of `info` blocks. Could you please move the note about TS being necessary up above the code snippet, and remove the `info` wrapper?